app/src/main/java/com/example/una/fragments/BroadcastsFragment.java

```diff
@@ -93,6 +93,7 @@ public void onSuccess(QuerySnapshot queryDocumentSnapshots) {
     }
 
     private void loadMoreData(QuerySnapshot documentSnapshots) {
+        adapter.clear();
```

Comment:
This will clear all the data while loading more pages. Is that expected?

Reply:
No, I didn't mean to clear all the data. When you first load the broadcasts page, it displays every broadcast twice so I was trying to fix that error